### DISCUSSION TITLE
Request not from admin network (flask.json) error

### DISCUSSION THREAD

**Discussion by SkyDragon2023** [aaronwmorris/indi-allsky#703] [Q&A] [answered]
I followed the guide to install the software, but I'm getting the above error when I try to generate a time-lapse, reboot or shut down the Pi. I'm logged in as the account created during install, this was done on a fresh install of the latest 64-bit raspberry OS. I have found that if I navigate to the Pi via the IP address I don't get the error, but if I use the DNS name I setup skypi.local the error is comes up when trying to generate time-lapse or shut down or reboot the Pi. Does Indi-allsky not support the use of DNS names or have I done something wrong?

**Comment by aaronwmorris** [comment, marked as answer]
I implemented a safety mechanism recently that prevents certain actions from being performed unless they originate from a trusted admin network.  The default admin network list is the usually private IP addresses.

Since data can be deleted from the web interface, I want to make extra sure these actions are controlled.

Since you using the .local address, it might be hitting your system via IPv6 and that will probably be outside the trusted networks.

You can check for messages in `/var/log/indi-allsky/webapp-indi-allsky.log`

**Reply by SkyDragon2023** [reply]
Many thanks, It would appear I still have things to learn.

**Reply by aaronwmorris** [reply]
No problem.  If you are on a private network, you can allow requests from all addresses by adding `0.0.0.0/0` for all IPv4 addresses and `::/0` for all IPv6 addresses.
